plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/model/CubridView.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.ext.cubrid.model;
+
+import java.util.Map;
+
+import org.jkiss.dbeaver.DBException;
+import org.jkiss.dbeaver.model.exec.jdbc.JDBCResultSet;
+import org.jkiss.dbeaver.model.meta.Property;
+import org.jkiss.dbeaver.model.runtime.DBRProgressMonitor;
+import org.jkiss.dbeaver.model.struct.DBSObjectWithScript;
+import org.jkiss.dbeaver.model.struct.rdb.DBSView;
+
+public class CubridView extends CubridTable implements DBSObjectWithScript, DBSView {
+
+	private String ddl;
+
+	public CubridView(CubridObjectContainer container, String tableName, String tableType, JDBCResultSet dbResult) {
+		super(container, tableName, tableType, dbResult);
+	}
+
+	@Override
+	public boolean isView() {
+		return true;
+	}
+
+	@Override
+	@Property(hidden = true, editable = true, updatable = true, order = -1)
+	public String getObjectDefinitionText(DBRProgressMonitor monitor, Map<String, Object> options) throws DBException {
+		return getDataSource().getMetaModel().getViewDDL(monitor, null, options);

```

Comment:
Does Cubrid have information about view definitions anywhere?

Reply:
It doesn't have  yet. We plan to add this feature in the next coming PR.